pkg/kv/dynamodb/store.go

```diff
@@ -45,18 +45,40 @@ const (
 	PartitionKey = "PartitionKey"
 	ItemKey      = "ItemKey"
 	ItemValue    = "ItemValue"
+
+	DefaultDynamoDBTableName = "kvstore"
+	// TODO (niro): Which values to use for DynamoDB tables?
+	DefaultDynamoDBReadCapacityUnits  = 1000
+	DefaultDynamoDBWriteCapacityUnits = 1000
 )
 
 //nolint:gochecknoinits
 func init() {
 	kv.Register(DriverName, &Driver{})
 }
 
+func normalizeDBParams(p *kvparams.DynamoDB) {
+	if len(p.TableName) == 0 {
+		p.TableName = DefaultDynamoDBTableName
+	}
+
+	if p.ReadCapacityUnits == 0 {
+		p.ReadCapacityUnits = DefaultDynamoDBReadCapacityUnits
+	}
+
+	if p.WriteCapacityUnits == 0 {
+		p.WriteCapacityUnits = DefaultDynamoDBWriteCapacityUnits
+	}
+}
+
 // Open - opens and returns a KV store over DynamoDB. This function creates the DB session
 // and sets up the KV table. dsn is a string with the DynamoDB endpoint
```

Comment:
```suggestion
// and sets up the KV table.
```
`dsn` seems irrelevant